Makefile

```
@@ -35,19 +38,16 @@ build-examples:
 
-licenses: .bin/licenses node_modules  # checks open-source licenses
+.PHONY: licenses
+licenses: .bin/licenses package-lock.json
 	.bin/licenses
 
-.PHONY: test
-test: install build-examples .bin/ory
-	./src/scripts/test.sh
-
 .bin/licenses: Makefile
 	curl https://raw.githubusercontent.com/ory/ci/master/licenses/install | sh
```

checkmake (0.3.2)

[warning] Target "curl https" should be declared PHONY.

(phonydeclared)